bentoml/yatai/deployment/sagemaker/serve

```diff
@@ -66,4 +66,14 @@ def _serve():
 
 
 if __name__ == '__main__':
+    container = BentoMLContainer()
+    config = BentoMLConfiguration()
+    if "BENTOML_GUNICORN_TIMEOUT" in os.environ:
+        config.override(["api_server", "port"], int(os.environ.get("BENTOML_GUNICORN_TIMEOUT")))
```

Comment:
is this a typo? `s/port/timeout`

Reply:
Good catch. This was a typo. We should think about adding unit test for this.

Reply:
Fixed.